renderdoc/driver/gl/gl_replay.cpp

```diff
@@ -206,9 +207,8 @@ void GLReplay::CreateOutputWindowBackbuffer(OutputWindow &outwin, bool depth)
   gl.glGenTextures(1, &outwin.BlitData.backbuffer);
   gl.glBindTexture(eGL_TEXTURE_2D, outwin.BlitData.backbuffer);
 
-  gl.glTextureImage2DEXT(outwin.BlitData.backbuffer, eGL_TEXTURE_2D, 0, eGL_SRGB8, outwin.width,
-                         outwin.height, 0, eGL_RGBA, eGL_UNSIGNED_BYTE, NULL);
-  gl.glTexParameteri(eGL_TEXTURE_2D, eGL_TEXTURE_MAX_LEVEL, 0);
+  gl.glTextureStorage2DEXT(outwin.BlitData.backbuffer, eGL_TEXTURE_2D, 1, eGL_SRGB8, outwin.width,
+                           outwin.height);
```

Comment:
What's the reason for changing this to `glTextureStorage2DEXT`? I recently changed this the other way specifically so that the replay could work even without ARB_texture_storage being available on desktop.